smoke_tests/processorFunctions.js

```diff
@@ -0,0 +1,104 @@
+const AWS = require('aws-sdk');
```

Comment:
Can you link to the docs & maybe add some comments in this file? It is quite hard to tell what it does otherwise, as how the Artillery scripting stuff works is not very obvious